doc/api/crypto.md

```diff
@@ -62,6 +62,7 @@ const cert2 = crypto.Certificate();
 <!-- YAML
 added: v0.11.8
 -->
+- `spkac` {string | Buffer}
```

Comment:
The sentence below:

```
The `spkac` argument can be either a string
 or a [`Buffer`][].
```

can be removed now.

Also, while you are at it, you can just add the return type here

```
* Returns {Buffer} The challenge component of the `spkac` data structure,
  which includes a public key and a challenge.
```

and remove the whole paragragh

```
 The `spkac` data structure includes a public key and a challenge. The
  `certificate.exportChallenge()` returns the challenge component in the
 form of a Node.js [`Buffer`][]. The `spkac` argument can be either a string
 or a [`Buffer`][].
```